index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FCC: Tribute Page</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main id="main">
+      <h1 id="title">Tribute Page</h1>
+      <p>Человек, который изменил мир.</p>
+      <div id="img-div">
+        <img id="image" src="stive.jpg" alt="Steve Jobs" />
+        <figure id="img-caption">
+          Я не доверяю компьютеру, который не могу поднять (Стив Джобс).
+        </figure>

```

Comment:
It is NOT the right way of using tag figure. Please, check out the documentation of this tag. Besides, there is a special tag for quotes.

Reply:
Corrected!